.github/workflows/ci.yaml

```diff
@@ -38,7 +38,7 @@ jobs:
       - name: Set up conda environment
         shell: bash -l {0}
         run: |
-          python -m pip install -e .
+          python -m pip install -e . --no-deps
```

Comment:
Do you think it would be good to have a 'pip-only' ci install (without corresponding conda yaml file), to check if our dependencies are up to date?

Reply:
With the new pip? Yep. For sure. We don't know yet how the solver will interact with our environments and packages.

Reply:
Ill set that up in a separate PR!